app/types/gasprice.go

```diff
@@ -11,7 +11,7 @@ const (
 
 	CongestionHigherGpMode = 0
 	NormalGpMode           = 1
```

Comment:
Does it have not used this variable NormalGpMode？

Reply:
Yes, there are only 3 dynamic gp modes, but the NormalGpMode constant has not been used in the code implementation, and it is used here for indication.